src/plugins/GUITestBase/GUITestBase.pro

```diff
@@ -259,6 +259,7 @@ HEADERS +=  src/GUITestBasePlugin.h \
             src/tests/common_scenarios/workflow_designer/estimating/GTTestsWorkflowEstimating.h \
             src/tests/common_scenarios/workflow_designer/name_filter/GTTestsWorkflowNameFilter.h \
             src/tests/common_scenarios/workflow_designer/parameters_validation/GTTestsWorkflowParameterValidation.h \
+            src/tests/common_scenarios/workflow_designer/samples/GTTestsWorkflowSamples.h \
```

Comment:
What kind of tests this file for? I'm not sure, that there is no way to add this test to an existing one, which is similar by purposes

Reply:
I don't know. I can move the test wherever you say
Probably this folder exists for tests 6940--6952